### DISCUSSION TITLE
마지막 보완점 정리

### DISCUSSION THREAD

**Discussion by Haroldfromk** [SwiftBreakers/TTEOPPOKKI4U#113] [General]
동익 : 
1. JSON 데이터 동기화
2. 프로필 수정페이지 닉네임 중복 확인 [완료]

**Comment by moremirim** [comment]
미림: 
1. 마이페이지 > 공지사항 내 내비게이션 탭바 아이템 "뒤로" 버튼의 간헐적 사라짐 이슈 해결
2. 유저별 리뷰 갯수에 따른 랭킹 부여 기능
3. 마이페이지 > 나의 찜 목록 > 북마크 내에서 북마크 아이콘 버튼을 통한 북마크 해제 기능

**Reply by jinmoon23** [reply]
게스트로 로그인 시 마이페이지에서 프로필수정 버튼이 없으면 좋을 것 같습니다!

**Reply by jinmoon23** [reply]
나의 찜 목록에서 북마크한 추천페이지 카드셀을 누르면 해당하는 카드셀의 DetailVC로 이동하면 좋을 것 같습니다!

**Reply by jinmoon23** [reply]
커뮤니티 닉네임중복방지 관련해서 에러핸들링하다가 발견했습니다!

프로필수정에서 닉네임을 완전히 비우고 저장을 하니 저장되었다는 alert은 뜨는데 정상적으로 수정이 안됩니다 빈 String으로! 만약 닉네임을 비울 수 없다면 그런 경우에 대해 기존과는 다른 alert을 띄워주면 좋을 것 같습니다!

**Reply by moremirim** [reply]
닉네임 중복방지 기능에서 empty string의 경우는 명일 duedate전에 업데이트 될 수 있도록 봐야겠습니다.

**Comment by labydin** [comment]
준영 : 
1. 가게 정보 페이지, 리뷰 보기 페이지 스크롤링 구현
2. back 버튼 아이콘 통일
3. 친구 찾기 버튼 연결

**Reply by jinmoon23** [reply]
지도페이지에서 스크랩 버튼을 누르면 추천페이지에서 북마크 눌렀을 때 뜨는 CustomAlert 띄우는 것도 좋을 것 같습니다! 어느정도의 통일성을 위해?!

**Reply by jinmoon23** [reply]
지도페이지에서 어노테이션핀 눌러서 들어가면 상단 이미지에 관련된 고민이 필요할 것 같습니다.

**Reply by labydin** [reply]
첫번째 피드백은 반영했습니다! 그러면서 VC에 tintcolor를 하얀색으로 추가했습니다. 이미지는 천천히 생각하는 걸로 ㅎㅎ

**Comment by jinmoon23** [comment]
진문 :

- [ ] 닉네임방지 기능 추가로 인해 닉네임 없는 사용자 입장 시 뜨던 alert 사라지는 버그 수정
- [x] 오픈톡 카메라 / 사진 / 지도 관련 에러핸들링
- [ ] 오픈톡 알람(자신이 속한 채널 한정)
- [ ] 로그인이 필요한 모든 경우 로그인페이지로 이동하도록 하는 Path 구현
